egs/wsj/s5/steps/score_kaldi.sh

```diff
@@ -137,9 +137,89 @@ if [ $stage -le 1 ]; then
   fi
 fi
 
+if $cer && [ $stage -le 2 ] ; then
```

Comment:
i mean move this block of code into a newly named script, but call it from this script.